spec/services/example_service_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
```

Comment:
I looks like this is missing the obnoxious `# frozen_string_literal: true`-comment

Reply:
Good catch. I just realized it's actually not enforced. Will open a new PR for it :-)